desktop/core/src/desktop/js/reactComponents/Table/Table.scss

```diff
@@ -0,0 +1,53 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'variables' as vars;
+@use 'mixins';
+
+.hue-table {
+  th {
+    background-color: vars.$fluidx-gray-040 !important;
```

Comment:
I would like to avoid using "!important" for a generic reusable component. Perhaps you could include the selectors from antd/cuix needed to make it work without?

Reply:
To play devils advocate, if this is s generic table component and not a "file table" component, why do we need a different background color than offered by the cuix? 

The same question could be asked for all changes, why do we need a Hue specific table that differs from the cuix default table?